ingress-controllers/readme.adoc

```diff
@@ -210,4 +210,325 @@ of the Ingress resource:
     $ kubectl delete -f templates/alb-ingress-controller.yaml
     $ kubectl delete -f https://raw.githubusercontent.com/coreos/alb-ingress-controller/master/examples/default-backend.yaml
 
-Check in the EC2 console to ensure your ALB has been deleted.
\ No newline at end of file
+Check in the EC2 console to ensure your ALB has been deleted.
+
+== Kube AWS Ingress Controller and Skipper
+
+link:https://github.com/zalando-incubator/kubernetes-on-aws[Kube AWS Ingress Controller]
+creates AWS Application Load Balancer (ALB) that is used to terminate TLS connections and use
+link:https://aws.amazon.com/certificate-manager/[AWS Certificate Manager (ACM)] or
+link:http://docs.aws.amazon.com/IAM/latest/APIReference/Welcome.html[AWS Identity and Access Management (IAM)]
+certificates. ALBs are used to route traffic to an Ingress http router for example
+link:https://github.com/zalando/skipper/[skipper], which routes
+traffic to Kubernetes services and implements
+link:https://zalando.github.io/skipper/dataclients/kubernetes/[advanced features]
+like green-blue deployments, feature toggles, reate limits, circuitbreakers, shadow traffic or A/B tests.
+
+In short the major differences from CoreOS ALB Ingress Controller is:
+
+- it uses Cloudformation instead of API calls
+- does not have routes limitations from AWS
+- automatically finds the best matching ACM and IAM certifacte for your ingress
+- you are free to use an http router imlementation of your choice, which can implement more features like green-blue deployments
+
+For this tutorial I assume you have GNU sed installed, if not read
+commands with `sed` to modify the files according to the `sed` command
+being run. If you are running BSD or MacOS you can use `gsed`.
+
+=== Create Kops cluster with cloud labels
+
+Cloud Labels are required to make Kube AWS Ingress Controller work,
+because it has to find the AWS Application Load Balancers it manages
+by AWS Tags, which are called cloud Labels in Kops.
+
+First you set some environment variables to choose AZs to deploy to,
+your S3 Bucket name for Kops configuration and you Kops cluster name:
+
+
+        export AWS_AVAILABILITY_ZONES=eu-central-1b,eu-central-1c
```

Comment:
`AWS_AVAILABILITY_ZONES` should already be setup as mentioned in pre-reqs. Can you state that accordingly?

Reply:
I hope this is now more clear